styleguide/_mixins.scss

```diff
@@ -9,3 +9,7 @@
     clear: both;
   }
 }
+
+@mixin kiln-hide {
```

Comment:
It could, but since it is tied to this test of the pattern used for the Kiln Pane, it might be best to keep it less global.